README.md

```diff
@@ -2,10 +2,14 @@
 
 [![Build Status](https://travis-ci.org/kubeapps/kubeapps.svg?branch=master)](https://travis-ci.org/kubeapps/kubeapps)
 
-Kubeapps is a set of tools written by [Bitnami](https://bitnami.com) to super-charge your Kubernetes cluster with:
+Kubeapps is a web-based UI for deploying and managing applications in Kubernetes clusters. Kubeapps allows you to:
 
-* Your own application [dashboard](https://kubeapps.com/), allowing you to deploy Kubernetes-ready applications into your cluster with a single click.
-* [Kubeless](http://kubeless.io/), a Kubernetes-native Serverless Framework, compatible with [serverless.com](https://serverless.com).
+- Browse and deploy [Helm](https://github.com/helm/helm) charts from chart repositories
+- Inspect, upgrade and delete Helm-based applications installed in the cluster
+- Add custom and private chart repositories (supports [ChartMuseum](https://github.com/helm/chartmuseum) and [JFrog Artifactory](https://www.jfrog.com/confluence/display/RTF/Helm+Chart+Repositories))
+- Browse and provision external services from the [Service Catalog](https://github.com/kubernetes-incubator/service-catalog) and available Service Brokers
+- Connect Helm-based applications to external services with Service Catalog Bindings
+- Securely manage applications in the cluster with [Role-Based Access Control](docs/user/access-control.md)
```

Comment:
This one seems to me more than an implementation detail than a feature.

Reply:
I'm referring to the fact that Kubeapps allows you to login with a user's token and therefore you can restrict certain actions to users (e.g. manage apps in a specific namespace). Is there a better way to mention that?

Reply:
So maybe it can be described as such?